Resources/Prototypes/_Moffstation/PersonalItems/Wearables/FrostWinters/Frost_Winters/frostmedicalwebbing.yml

```
@@ -0,0 +1,23 @@
+# Player FrostWinters - Character: Frost Winters
+
+- type: entity
+  parent: ClothingBeltMedicalFilled
+  id: PersonalItemFrostMedicalWebbing
+  name: "Winters' medical rig"
+  description: "Winters' chest rig brought from home, modified to fit medical supplies. Looks oddly familiar to some with certain backgrounds."
+  suffix: PersonalItem, Filled
+  components:
+  - type: Sprite
+    sprite: _Moffstation/PersonalItems/Wearables/FrostWinters/Frost_Winters/frostmedicalwebbing.rsi
+  - type: Clothing
+    sprite: _Moffstation/PersonalItems/Wearables/FrostWinters/Frost_Winters/frostmedicalwebbing.rsi
+
+- type: loadout
+  id: PersonalItemFrostMedicalWebbing
+  storage:
+    belt:
+    - PersonalItemFrostMedicalWebbing
```

Just noticed this, but this will try to store the webbing in the item in the belt slot.
Should be
```suggestion
equipment:
  belt: PersonalItemFrostMedicalWebbing
```
but I'll be fixing it with the loadout override in like two seconds, so this is just an FYI